src/libs/actions/Report.ts

```diff
@@ -1865,7 +1857,7 @@ function addEmojiReaction(reportID, reportActionID, emoji, skinTone = preferredS
                     users: {
                         [currentUserAccountID]: {
                             skinTones: {
-                                [!_.isUndefined(skinTone) ? skinTone : -1]: createdAt,
+                                [skinTone ?? -1]: createdAt,
```

Comment:
Not sure if we want to change logic here, but if I remember correctly there's a default skin tone defined in CONST, maybe we could use it here instead of `-1`?